### PR TITLE
raider: init at 1.3.1

### PR DESCRIPTION
**Pull request by BenediktBroich**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/raider/default.nix [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  ];
+
+  postInstall = ''
+    ${glib.dev}/bin/glib-compile-schemas $out/share/glib-2.0/schemas
+  '';
+
+  meta = with lib; {
+    description = "Securely delete your files";
+    homepage = "https://apps.gnome.org/de/app/com.github.ADBeveridge.Raider";
+    license = licenses.gpl3;
```
Comment (jtojnar) [inline pkgs/applications/misc/raider/default.nix:52]: `gpl3` is unclear, use `gpl3Plus` or `gpl3Only`. See https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206 for more details.

File: pkgs/applications/misc/raider/default.nix [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    ninja
+    pkg-config
+    wrapGAppsHook4
+  ];
+
+  buildInputs = [
+    gtk4
+    libadwaita
+  ] ++ lib.optionals (!stdenv.buildPlatform.canExecute stdenv.hostPlatform) [
+    mesonEmulatorHook
```
Comment (jtojnar) [inline pkgs/applications/misc/raider/default.nix:42]: Should not this go to `nativeBuildInputs`?

File: pkgs/applications/misc/raider/default.nix [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+, stdenv
+, wrapGAppsHook4
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "raider";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "ADBeveridge";
+    repo = finalAttrs.pname;
```
Comment (jtojnar) [inline pkgs/applications/misc/raider/default.nix:23]: It probably does not make much sense to couple the repo name to `pname`. If `pname` is changed in an overlay, we would not want to break the source URL.

File: pkgs/applications/misc/raider/default.nix (hunk 1 above)
Comment (bjornfor) [reply pkgs/applications/misc/raider/default.nix:52]: > homepage = "https://apps.gnome.org/de/app/com.github.ADBeveridge.Raider";

It's a bad URL: `We are sorry. But the page you are looking for is not available.`

File: pkgs/applications/misc/raider/default.nix (hunk 2 above)
Comment (BenediktBroich) [reply pkgs/applications/misc/raider/default.nix]: I can try. I don't rally know. Found it in a comment somewhere.

File: pkgs/applications/misc/raider/default.nix [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    libadwaita
+  ];
+
+  postInstall = ''
+    ${glib.dev}/bin/glib-compile-schemas $out/share/glib-2.0/schemas
+  '';
+
+  meta = with lib; {
+    description = "Securely delete your files";
+    homepage = "https://apps.gnome.org/de/app/com.github.ADBeveridge.Raider";
```
Comment (jtojnar) [inline pkgs/applications/misc/raider/default.nix:53]: This redirects me to `https://apps.gnome.org/en-GB/de/app/com.github.ADBeveridge.Raider/`, which does not exist.

File: pkgs/applications/misc/raider/default.nix (hunk 3 above)
Comment (jtojnar) [reply pkgs/applications/misc/raider/default.nix]: Using `finalAttrs` for version was fine. I just meant using `"raider"` here literally.

File: pkgs/applications/misc/raider/default.nix (hunk 4 above)
Comment (BenediktBroich) [reply pkgs/applications/misc/raider/default.nix:53]: The de link works for me. Is [this](https://apps.gnome.org/app/com.github.ADBeveridge.Raider/) link working for you?

File: pkgs/applications/misc/raider/default.nix [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    mesonEmulatorHook
+  ];
+
+  buildInputs = [
+    gtk4
+    libadwaita
+  ];
+
+  postInstall = ''
+    ${glib.dev}/bin/glib-compile-schemas $out/share/glib-2.0/schemas
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/raider/default.nix:48]:
```suggestion
    glib-compile-schemas $out/share/glib-2.0/schemas
```
please rely on nativeBuildInputs

File: pkgs/applications/misc/raider/default.nix (hunk 4 above)
Comment (SuperSandro2000) [reply pkgs/applications/misc/raider/default.nix:53]: https://apps.gnome.org/app/com.github.ADBeveridge.Raider/ works

File: pkgs/applications/misc/raider/default.nix (hunk 3 above)
Comment (BenediktBroich) [reply pkgs/applications/misc/raider/default.nix]: Yes. I needed finalAttrs for the version test. But `raider --version` does not return a proper version. So i removed it.

**Pull request merged**